google-map-marker.html

```diff
@@ -366,7 +366,7 @@
         subtree: true
       });
 
-      var content = this.innerHTML.trim();
+      var content = this.children[0];
```

Comment:
This is selecting the first light dom child. are you meaning to pull the `<div>` in this element's shadow dom? I think this will work b/c of shady dom but fail under native shadow dom.